core/src/test/scala/parseback/CompoundParserSpec.scala

```diff
@@ -16,7 +16,7 @@
 
 package parseback
 
-object CompoundParserSpec extends ParsebackSpec {
+class CompoundParserSpec extends ParsebackSpec {
```

Comment:
Specs weren't running. Switching them from objects to classes seems to have done the trick.

Here's the issue I found that led me to that conclusion: https://github.com/etorreborre/specs2/issues/757